Book sugestions

Drocheg opened the pull request:
Adds book sugestion model

Drocheg committed:
Adds book sugestion model

Drocheg committed:
Makes bookSugestion dependent destroy with User

Drocheg commented:
https://trello.com/c/Fz8Ja2Ug/29-crear-modelo-de-booksuggestion

Gmarn commented on app/models/book_sugestion.rb:
```diff
@@ -0,0 +1,7 @@
+class BookSugestion < ApplicationRecord
+  validates :author, :title, :link, presence: true
+
+  has_one :user, dependent: :destroy
```
Are you sure you want a  `dependent: :destroy`? 
[Here](https://guides.rubyonrails.org/association_basics.html) some help.

Gmarn commented on db/migrate/20180910185811_create_book_sugestions.rb:
```diff
@@ -0,0 +1,15 @@
+      t.string :link, null: false
+      t.string :publisher
+      t.string :year
+      t.references :user, foreign_key: true
```
Were you able to create a boo suggestion without an user?

Drocheg replied:
In rails console, yes.

Drocheg committed:
Changes user reference in book sugestion to nullify from destroy

Drocheg committed:
Book sugestions route (#18)

-Adds bookSugestion controller and create route
-Adds user that made sugestion when user logged in
-Skips authentication for book sugestion creation
